packages/account/imAccount/index.test.ts

```diff
@@ -0,0 +1,88 @@
+import * as ethers from "ethers"
+
+import config from "~config/test"
+import { ECDSAValidator } from "~packages/account/imAccount/validator/ecdsaValidator"
+import { WaalletBackgroundProvider } from "~packages/provider/waallet/background/provider"
+import { WaalletRpcMethod } from "~packages/provider/waallet/rpc"
+import number from "~packages/util/number"
+import { describeAccountSuite } from "~packages/util/testing/suite/account"
+import type { HexString } from "~typing"
+
+import { imAccount } from "./index"
+
+async function setNewECDSAOwner(
+  provider: WaalletBackgroundProvider,
+  ecdsaValidator: ECDSAValidator,
+  newOwnerAddress: string
+) {
+  await provider.request<HexString>({
+    method: WaalletRpcMethod.eth_sendTransaction,
+    params: [
+      {
+        to: await ecdsaValidator.getAddress(),
+        data: ecdsaValidator.getSetOwnerCallData(newOwnerAddress)
+      }
+    ]
+  })
+}
+
+describeAccountSuite(
+  "imAccount",
+  () => {
+    const ecdsaValidator = new ECDSAValidator({
+      address: config.address.ECDSAValidator,
+      ownerPrivateKey: config.account.operator.privateKey,
+      nodeRpcUrl: config.rpc.node
+    })
+
+    return imAccount.initWithFactory({
+      factoryAddress: config.address.imAccountFactory,
+      implementationAddress: config.address.imAccountImplementation,
+      entryPointAddress: config.address.EntryPoint,
+      validator: ecdsaValidator,
+      fallbackHandlerAddress: config.address.FallbackHandler,
+      salt: number.random(),
+      nodeRpcUrl: config.rpc.node
+    })
+  },
+  (ctx) => {
+    // TODO: This test at this moment relies on tests in test bed to deploy the account.
+    // It would be better to decouple it.
+    it("should init with existing imAccount", async () => {
+      const a = await imAccount.init({
+        address: await ctx.account.getAddress(),
+        nodeRpcUrl: config.rpc.node,
+        validator: ctx.account.validator,
+        entryPointAddress: config.address.EntryPoint
+      })
+
+      expect(await a.getAddress()).toBe(await ctx.account.getAddress())
+    })
+
+    // Set new ECDSAValidator owner
+    it("should change the owner of ECDSAValidator", async () => {
+      const ecdsaValidator = ctx.account.validator as ECDSAValidator
+      const newOwner = new ethers.Wallet(
+        "0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff81",
+        new ethers.JsonRpcProvider(config.rpc.node)
+      )
```

Comment:
I found this owner is same as the one in config. Maybe here we can generate a random wallet through `Wallet.createRandom`

Reply:
Oh, I modify the final character haha! I tried the `createRandom` before, but the return type `[HDNodeWallet](https://docs.ethers.org/v6/api/wallet/#HDNodeWallet)` need to be transformed. If transforming is fine, I can use `createRandom`!